ingest/profiles/default/config.yaml

```diff
@@ -0,0 +1,4 @@
+cores: all
+rerun-incomplete: true
+printshellcmds: true
+reason: true
```

Comment:
These options are newly introduced, but seems reasonable to me.

Reply:
Yep, rerun incomplete just avoids an error, it's something almost everyone adds all the time anyways when running builds